src/components/Proposals/ProposalDetails/ProposalDetails.js

```diff
@@ -104,6 +104,14 @@ function ProposalDetails({ vote }) {
               vote={vote}
               disabledProgressBars={disabledProgressBars}
             />
+            {youVoted && (
+              <VoteCast
+                accountVote={voterInfo.hasVoted}
```

Comment:
I think it would be simpler to make this prop a boolean and pass `accountHasVoted` with a matching prop name.